src/plots/VolumeVTK9/PyVolumeAttributes.C

```diff
@@ -3459,43 +3459,38 @@ PyVolumeAttributes_getattr(PyObject *self, char *name)
 #include <visit-config.h>
 
 #if VISIT_OBSOLETE_AT_VERSION(3,5,0)
-#error This code is obsolete in this verison of VisIt and should be removed.
+#error This code is obsolete in this version of VisIt and should be removed.
 #else
     // Try and handle legacy fields
-#define DEPRECATED_MESSAGE(type, msg) \
-    PyErr_WarnFormat(NULL, 1, "%s is no longer a valid Volume attribute.\n" \
-                    "It's value is being ignored, " \
-                    "please remove it from your script.\n", \
-                    "Try using %s instead.\n", type, msg);
-#define NAME_CHANGE_MESSAGE(old, new) \
-    PyErr_WarnFormat(NULL, 1, "%s is no longer a valid Volume attribute.\n" \
-                    "It's name has been changed to %s, " \
-                    "please update your script.\n", old, new);
+#define NAME_CHANGE_MESSAGE2(oldname, newname) \
```

Comment:
I notice you deleted `NAME_CHANGE_MESSAGE` and added `NAME_CHANGE_MESSAGE2`. Did you want to rename it back to the name without the `2`? I'm fine either way, but I thought I'd mention it.

Reply:
There is another #define NAME_CHANGE_MESSAGE in the same source file, got compiler warnings, so I changed the name of one.   I believe the multiple defines are needed due to how we add mods to python atts via xml2python and the .code file. One is defined for getattr and the other for setattr.

Reply:
Sounds good!